### DISCUSSION TITLE
Imported Process configs not showing up

### DISCUSSION THREAD

**Discussion by cjourdan87** [OrcaSlicer/OrcaSlicer#2098] [General]
Using Windows 11, version 1.7.0 beta for my SV07+.  I downloaded some profiles others have made.  I can initially add the process config files and use them, but when if close Orca then re-open it, the imported process does not show up in the list.  
When I go to import the same process, I get a notice saying a config with the same name already exists.  If I hit 'Yes' to overwrite, it says one config was added, but doesn't show in the list.  If I edit and save a copy of one of the default processes, it seems to show up everytime.

I used the .exe version and the .zip version with similar results.
I switch to 1.6.6 and all of the imported configs showed up.
I deleted the OrcaSlicer folder under AppData folder to 'reset' Orca and getting the same results.
Any ideas?

![1](https://github.com/SoftFever/OrcaSlicer/assets/122924013/a37ed728-92ba-4113-8da1-74b4acf772be)

![4](https://github.com/SoftFever/OrcaSlicer/assets/122924013/d311bb42-9694-4e0a-b01c-49fea7fd2205)

**Comment by cjourdan87** [comment]
I just imported a different file, it showed up, but when I switched to a different process, it disappeared from the list and its the same thing with that config of it saying it already exists.

**Comment by theophile** [comment]
Check your `compatible_print_conditions` and `compatible_filament_conditions` strings. If in doubt, delete them and see if that resolves the problem.

**Reply by cjourdan87** [reply]
I apologize if the answer is obvious, but how/where do I check that?

**Reply by theophile** [reply]
Open the .json files in a text editor and look for those "conditions" keys. Ctrl+F to search for the word condition.

**Reply by cjourdan87** [reply]
Checking the conditions in the text editor, I only saw compatible_printers_condition, but right above that there was a compatible_printers and it said "Sovol SV06 Plus 0.4 nozzle".  I changed it to 07 and imported and overwrote the config files and it appears to be working properly now.

Thank you and I appreciate the help with this!
